apps/settings/index.html

```diff
@@ -59,6 +59,8 @@
     <link rel="stylesheet" type="text/css" href="/shared/elements/gaia-icons/gaia-icons.css">
     <link rel="stylesheet" type="text/css" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css">
     <script src="shared/elements/gaia-header/dist/gaia-header.js"></script>
+    <script defer src="/shared/js/component_utils.js"></script>
```

Comment:
nit: Please help remove this line[[1]] that loads the same file.

[1]: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/modules/panel_cache.js#L26